lisa/sut_orchestrator/libvirt/schema.py

```diff
@@ -133,10 +133,15 @@ def __post_init__(self) -> None:
             self.disk_img_format = DiskImageFormat.QCOW2.value
 
 
+@dataclass_json()
+@dataclass
+class KernelSchema:
+    path: str = ""
+    is_remote_path: bool = False
+
+
 @dataclass_json()
 @dataclass
 class CloudHypervisorNodeSchema(BaseLibvirtNodeSchema):
-    # Local path to the cloud-hypervisor firmware.
-    # Can be obatained from:
-    # https://github.com/cloud-hypervisor/rust-hypervisor-firmware
-    firmware: str = ""
+    # Local or remote path to the cloud-hypervisor kernel.
+    kernel: Optional[KernelSchema] = None
```

Comment:
This is a backwards incompatible schema change. I'm fine with it but we need to make sure to update our runbooks as soon as the new version of LISA lands.

Another option is to support both fields for now and remove `firmware` in a later release.

cc: @pupacha @smit-gardhariya

Reply:
Yes, it's a breaking change, we can have both `firmware` and `kernel` in runbook schema for now.

Reply:
@gamora12 @anirudhrb  Since the schema change is not so complex, we can update our runbooks with the new LISA release.

Reply:
Just discussed offline with @pupacha. Summarizing it here:

It is not about the schema change being complex or simple. It is about the urgency it creates to update the runbooks immediately after new release. It could happen that we might miss it and all the pipelines will start failing. I guess this is what we have been doing in the past. However, it is a good practice in general to make backwards compatible changes and eventually deprecate the old interface. So, let's start practicing that starting with this change.

Reply:
Sure, we should continue this practice. We can remove the firmware parameter in the next PR.